app/models/lettings_log.rb

```diff
@@ -149,7 +149,7 @@ def weekly_value(field_value)
   def weekly_to_value_per_period(field_value)
     num_of_weeks = NUM_OF_WEEKS_FROM_PERIOD[period]
 
-    ((field_value * 52) / num_of_weeks).round(2)
+    sprintf("%.2f", (field_value * 52) / num_of_weeks)
```

Comment:
do you really want to be converting these to strings at this stage?

Reply:
is this just taking a number and formatting it right?
We have things that do this already I believe
eg `log:format_as_currency` (if using this you could change its parameter from num_string to num)
if that doesn't work for you because you don't want the '£' then you could use rails `number_with_delimeter` or `number_with_precision`